GameData/RealismOverhaul/RO_SuggestedMods/AviationCockpits/Cockpits.cfg

```diff
@@ -86,75 +88,246 @@
 	}
 }
 
+// Focke Wulf 190 radial cockpit
+@PART[Focke-Wulf?Bearcat?Cockpit]:FOR[RealismOverhaul]
+{
+	%RSSROConfig = True
+	@title = FW-190 Cockpit (Radial)
+	@manufacturer = Focke-Wulf
+	@description = Radially-attachable, single seat fighter cockpit.
+	%rescaleFactor = 1.16 // same as Meteor
+
+	@mass = 0.25 // mass of Meteor minus mass of Mk1 structural fuselage
+	@crashTolerance = 8
+	@breakingForce = 2000
+	@breakingTorque = 2000
+	%maxTemp = 350
+	%skinMaxTemp = 450
+
+	@MODULE[ModuleCommand]
+	{
+		@RESOURCE
+		{
+			name = ElectricCharge
+			rate = 2.0
+		}
+	}
+	@RESOURCE[ElectricCharge]
+	{
+		@amount = 7200
+		@maxAmount = 7200
+	}
+	RESOURCE
+	{
+		name = Oxygen
+		amount = 15
+		maxAmount = 15
+	}
+	RESOURCE
+	{
+		name = Water
+		amount = 2
+		maxAmount = 2
+	}
+	RESOURCE
+	{
+		name = Food
+		amount = 1
+		maxAmount = 1
+	}
+	!MODULE[ModuleReactionWheel] {}
+	MODULE
+	{
+		name = ModuleFuelTanks
+		type = Structural
+		basemass = -1
+		volume = 20 //
+	}
+	MODULE
+	{
+		name = ModuleSAS
+	}
+	MODULE
+	{
+		name = ModuleKrEjectPilot
+		MODULE
+		{
+			name = ModuleKrKerbalParachute
+			deployedDrag = 100
+			minAirPressureToOpen = 0.01
+			semiDeployedFraction = 0.0025
+			semiDeployedHeight = 1.25
+			deployTime = 0.33
+		}
+	}
+}
+@INTERNAL[Fw_190_Internal]:FOR[RealismOverhaul]
+{
+	%scaleAll = 1.16, 1.16, 1.16
+	@MODULE[InternalSeat],*
+	{
+		%kerbalScale = 1.16, 1.16, 1.16
+		%kerbalOffset = 0.0, 0.0, 0.0
+		%kerbalEyeOffset = 0.0, 0.0, 0.0
+	}
+}
+
+// Bearcat radial cockpit
+@PART[F8f?Bearcat?Cockpit]:FOR[RealismOverhaul]
+{
+	%RSSROConfig = True
+	@title = F8F Bearcat Cockpit (Radial)
+	@manufacturer = Grumman
+	@description = Radially-attachable, single seat fighter cockpit.
+	%rescaleFactor = 1.16
+
+	@mass = 0.25
+	@crashTolerance = 8
+	@breakingForce = 2000
+	@breakingTorque = 2000
+	%maxTemp = 350
+	%skinMaxTemp = 450
+
+	@MODULE[ModuleCommand]
+	{
+		@RESOURCE
+		{
+			name = ElectricCharge
+			rate = 2.0
+		}
+	}
+	@RESOURCE[ElectricCharge]
+	{
+		@amount = 7200
+		@maxAmount = 7200
+	}
+	RESOURCE
+	{
+		name = Oxygen
+		amount = 15
+		maxAmount = 15
+	}
+	RESOURCE
+	{
+		name = Water
+		amount = 2
+		maxAmount = 2
+	}
+	RESOURCE
+	{
+		name = Food
+		amount = 1
+		maxAmount = 1
+	}
+	!MODULE[ModuleReactionWheel] {}
+	MODULE
+	{
+		name = ModuleFuelTanks
+		type = Structural
+		basemass = -1
+		volume = 20 //
```

Comment:
I based the new configs on the original config @NathanKell had written for the Interceptor?Cockpit. I can delete the Resources and put them as TANKS within the ModuleFuelTanks, no probs.